src/worker.py

```
@@ -2708,11 +2708,67 @@ async def _dispatch(request, env):
+
         return err("API endpoint not found", 404)
 
     return await serve_static(path, env)
 
 
+# Feedback API
+
+async def api_submit_feedback(request, env):
+    try:
+        body = json.loads(await request.text())
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)